pkg/driver/driver_alicloud.go

```diff
@@ -75,6 +76,11 @@ func (c *AlicloudDriver) Create() (string, string, error) {
 		request.InternetMaxBandwidthOut = requests.NewInteger(*c.AlicloudMachineClass.Spec.InternetMaxBandwidthOut)
 	}
 
+	if ok := c.AlicloudMachineClass.Spec.DataDisks != nil; ok {
```

Comment:
Should this line be replaced with just:
`c.AlicloudMachineClass.Spec.DataDisks != nil`
Also, since we are doing a `nil` check, do we need the following check also:
`len(c.AlicloudMachineClass.Spec.DataDisks) > 0`

Reply:
both checks added